README.md

```diff
@@ -197,7 +197,12 @@ In order to change between subdrivers use the [driver](#settings-api) setting. S
 - getPageSource: The returned page source is retrieved from Compose and all elements there contain [Compose-specific](#compose-element-attributes) attributes.
 - click, isDisplayed, isEnabled, clear, getText, sendKeys, getElementRect, getValue, isSelected: These commands should properly support compose elements.
 - getAttribute: Accepts and returns Compose-specific element attributes. See [Compose Element Attributes](#compose-element-attributes) for the full list of supported Compose element attributes.
-
+- mobile: swipe command should properly support compose element as long as the element is swipeable.
```

Comment:
it is not necessary to repeat arguments here as the API is already documented elsewhere. Just mention it is supported